StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml

```diff
@@ -1,9 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="51c8f867-9fa0-4ca8-a82f-0f76d06d3bc2" />
-  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-ControlledDrug" />
-  <name value="ExtensionDMControlledDrug" />
+  <id value="Extension-England-DMControlledDrug" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMControlledDrug" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandDMControlledDrug" />
+  <title value="Extension England DM ControlledDru" />
   <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
```

Comment:
Duplicate contact

Reply:
Removed Duplicate Contact from this file.